src/js/lib/client/src/index.js

```diff
@@ -383,6 +382,15 @@ function urlParamsFromObject(obj) {
 function createWindow() {
     let webPreferences     = secureWebPreferences()
     webPreferences.preload = path.join(root,'preload.js')
+
+    // Note: this is a conditional default. We want borders on Win/Linux and borderless on Mac.
```

Comment:
Describe why! We want borderless everywhere, but there are bugs that prevent us from doing it. This should be documented. Moreover, it should be reported in electron repo and linked here.